assets/js/modules/pagespeed-insights/components/DashboardPageSpeedWidget.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Dashboard PageSpeed Widget component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { string } from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_STORE } from '../../../googlesitekit/modules/datastore/constants';
+import DashboardPageSpeedCTA from './DashboardPageSpeedCTA';
+import DashboardPageSpeed from './DashboardPageSpeed';
+const { useSelect } = Data;
+
+function DashboardPageSpeedWidget( { width, slug } ) {
+	const pagespeedInsightsModule = useSelect( ( select ) => select( MODULES_STORE ).getModule( 'pagespeed-insights' ) );
+	if ( ! pagespeedInsightsModule ) {
+		return null;
+	}
+
+	const { active, setupComplete } = pagespeedInsightsModule;
+	if ( ! active && ! setupComplete ) {
+		return <DashboardPageSpeedCTA />;
+	}
+
+	const { Widget } = global.googlesitekit.widgets.components;
```

Comment:
We should import these with normal imports in our own code rather than using the global. (eg `import Widget from '../$WIDGET_PATH`;`

I just realised the IB mentions using the global directly, but we should nix that and import this component directly in our own code. Not sure why the IB said to do it this way… sorry about that.

Reply:
Updated.